src/api-spec/payment-token-adapter-spec.yaml

```diff
@@ -58,6 +60,523 @@ paths:
             type: string
       responses:
         '200':
-          description: ''
-tags: []
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  dateOfBirth:
+                    type: string
+                    description: 'Date Of Birth'
+                  displayName:
+                    type: string
+                    description: 'Display Name'
+                  extensionList:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        key:
+                          type: string
+                        value:
+                          type: string
+                    description: "Extension List"
+                  firstName:
+                    type: string
+                    description: 'First Name'
+                  fspId:
+                    type: string
+                    description: 'FSP Id'
+                  idSubValue:
+                    type: string
+                    description: 'Id Sub Value'
+                  idType:
+                    type: string
+                    description: 'Type of Id'
+                  idValue:
+                    type: string
+                    description: 'ID Value'
+                  lastName:
+                    type: string
+                  merchantClassificationCode:
+                    type: string
+                  middleName:
+                    type: string
+                  type:
+                    type: string
+                  supportedCurrencies:
+                    type: array
+                    items:
+                      type: string
+                  kycInformation:
+                    type: string
+
+          description: 'Response containing details of the requested party'
+      tags:
+        - Discovery
+
+  /quoterequests:
```

Comment:
I think this swagger should be split into two. One for connector service and another for token service.

Reply:
The issue is some of these routes are shared.